### PR TITLE
DT-897 remove admin functions

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: src/KeyworkerManagement/App.js [hunk 1]
```diff
@@ -272,42 +252,31 @@ class App extends React.Component {
...
-              path="/manage-key-workers/key-worker-search/results"
+              path="/key-worker-search/results"
               render={() => (
                 <KeyworkerSearchResultsContainer displayBack={this.displayBack} handleError={this.handleError} />
               )}
             />
             <Route
               exact
-              path="/manage-key-workers/key-worker/:staffId"
+              path="/key-worker/:staffId"
```
Comment (Nimphal) [inline src/KeyworkerManagement/App.js:262]: New-nomis-ui points to this url, and there may be other apps. Have you tracked them all or perhaps maybe redirects?

Comment (mikehalmamoj) [reply src/KeyworkerManagement/App.js:262]: I was aware of new-nomis-ui but not others... But a redirect solves my problem of release timings.  Thanks!

File: src/KeyworkerManagement/App.js [hunk 2]
```diff
@@ -179,21 +168,13 @@ class App extends React.Component {
...
-
     let innerContent
     const routes = (
       // eslint-disable-next-line
       <div className="inner-content" onClick={() => boundSetMenuOpen(false)}>
         <div className="pure-g">
           <Switch>
-            {hasAdminUtilities && <Route exact path="/" render={() => <Redirect to="/admin-utilities" />} />}
-            {hasKWRoles && <Route exact path="/" render={() => <Redirect to="/manage-key-workers" />} />}
-            {!(hasKWRoles || hasAdminUtilities) && (
```
Comment (petergphillips) [inline src/KeyworkerManagement/App.js:194]: this doesn't look right - there is no condition on `/unauthorised` any more?

Comment (mikehalmamoj) [reply src/KeyworkerManagement/App.js:194]: That code was there to route you to the right page when landing on `/` but with only one of the authorities.  It didn't actually stop you from typing in `/manage-key-workers` to the address bar and so it wasn't really adding any security.

**Pull request merged**